resources/quality/volumic/stream30ultra_high.inst.cfg

```diff
@@ -0,0 +1,30 @@
+[general]
+version = 4
+name = Très fin
```

Comment:
Names should be in English.

Reply:
all names have been translated in english